src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,10 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  movies.map(movie => (
+    <div key={movie.imdbId} className="movies">
+      <MovieCard movie={movie} />
+    </div>
+  ))
```

Comment:
Try to refactor this part. The container with class `.movies` should be only 1, and it should contain all movies inside it